libs/platform/src/lib/table-helpers/directives/table-virtual-scroll.directive.ts

```diff
@@ -129,8 +129,9 @@ export class TableVirtualScrollDirective extends TableVirtualScroll implements O
                 filter(() => this.virtualScroll && !!this.bodyHeight),
                 takeUntil(this._destroy$)
             )
-            .subscribe(() => {
+            .subscribe((scrollable) => {
                 this.calculateVirtualScrollRows();
+                this._table.getTableState().scrollTopPosition = scrollable.getScrollTop();
```

Comment:
use `setTableState` instead

Reply:
ok. I assume you mean use setTableState and set this value using spread operator.